types/office-js-preview/index.d.ts

```diff
@@ -15083,6 +15083,11 @@ declare namespace Office {
          *
          * - `NumberOfAttachmentsExceeded`: The message or appointment has too many attachments.
          *
+         * **Note**: If you're adding an inline base64 image to the body of a message being composed, you must first get the current message body using the
+         * {@link https://docs.microsoft.com/javascript/api/outlook/office.body#outlook-office-body-getasync-member(1) | Office.context.mailbox.item.body.getAsync} 
+         * method before inserting the image using the `addFileAttachmentFromBase64Async` method. Otherwise, the image will not render in the message once it's inserted.
```

Comment:
```suggestion
         * method before inserting the image using `addFileAttachmentFromBase64Async`. Otherwise, the image will not render in the message once it's inserted.
```